app/javascript/pages/browse/ListBrowser.vue

```diff
@@ -30,6 +31,7 @@
         <td>
           <a :href="contribution.respond_path" class="button icon-list is-primary"><span class=""> Respond</span></a>
         </td>
+        <td>{{ contribution.name.match(/\((.*)\)/)[1] }}</td>
```

Comment:
`/\((.*)\)/` this regex is identical with the one below. I think you should use some constant to config it somewhere, then use it for both places.

Next time when we want to change this regex, we just need to do it once.

Reply:
That's a good idea - however I didn't end up merging this PR (see the discussion of the feature above). So I won't be refactoring further for now as this code isn't in main.

@svileshina has taken over work on this feature on https://github.com/rubyforgood/mutual-aid/pull/917